config/main.py

```diff
@@ -2681,6 +2681,254 @@ def remove(ctx, interface_name, ip_addr):
         ctx.fail("'ip_addr' is not valid.")
 
 
+#
+# Use this method to validate unicast IPv4 address
+#
+def is_ip4_addr_valid(addr, display):
+    v4_invalid_list = [ipaddress.IPv4Address(unicode('0.0.0.0')), ipaddress.IPv4Address(unicode('255.255.255.255'))]
```

Comment:
Conversion to `unicode()` should not be necessary, now that sonic-utilities is Python 3-only. This may also be causing the PR test failures.

Reply:
Done. Removed unicode function.